README.md

```diff
@@ -77,13 +77,37 @@ Run `make ocs-registry` to generate the registry bundle container image.
 
 ## Prerequisites
 
-+ OCS Operator will install its components only on nodes marked for OCS (i.e. nodes with label `cluster.ocs.openshift.io/openshift-storage=''`)
+OCS Operator will install its components only on nodes labelled for OCS with the key `cluster.ocs.openshift.io/openshift-storage=''`.
+To label the nodes from CLI,
 
-  To label the nodes from CLI,
+```console
+$ oc label nodes <NodeName> cluster.ocs.openshift.io/openshift-storage=''
+```
+
+OCS requires at least 3 labelled nodes.
+
+When creating StorageCluster from the UI, the create wizard takes care of labelling the selected nodes.
+
+OCS requires at least 3 nodes labelled this way.
+
+### Dedicated nodes
+
+In case dedicated storage nodes are available, these can also be tainted to allow only OCS components to be scheduled on them.
+Nodes need to be tainted with `node.ocs.openshift.io/storage=true:NoSchedule` which can done from the CLI as follows,
+
+```console
+$ oc adm taint nodes <NodeNames> node.ocs.openshift.io/storage=true:NoSchedule
+```
+
+> Note: Not all OCS created resources have their tolerations set currently.
+> When working with a minimum sized cluster with 3 worker nodes, tainting all 3
+> nodes will cause failure of OCS to fully deploy.
+> As a workaround, an extra worker node which is not tainted with the above
+> taint can be created.
```

Comment:
As explained above, I would not call this a workaround. It's just a requirement if we do taint.